src/selection-handler.js

```diff
@@ -260,32 +262,45 @@ Object.assign(SelectionHandler.prototype, require('./function-bind'), require('.
     var ctrlKey = ev.ctrlKey || ev.metaKey;
     var key = ev.key;
 
-    if (ctrlKey && key === "a") {
+    if (this.selecting && key === "Backspace") {
+      ev.preventDefault();
+      this.delete();
+    } else if (ctrlKey && key === "a") {
       if (!this.selecting && this.cancelSelectAll()) return;
       ev.preventDefault();
       this.mediator.trigger("selection:all");
-    } else if (this.selecting && ["Down", "ArrowDown"].indexOf(key) > -1) {
-      ev.preventDefault();
-      if (ev.shiftKey && ev.altKey) this.expandToEnd();
-      else if (ev.shiftKey) this.expand(1);
-      else if (ev.altKey) this.startAtEnd();
-      else {
-        this.cancel();
-        this.mediator.trigger("block:focusNext", this.getEndBlock().blockID, { force: true });
-        return;
-      }
-      this.focusAtEnd();
-    } else if (this.selecting && ["Up", "ArrowUp"].indexOf(key) > -1) {
-      ev.preventDefault();
-      if (ev.shiftKey && ev.altKey) this.expandToStart();
-      else if (ev.shiftKey) this.expand(-1);
-      else if (ev.altKey) this.start(0);
-      else {
-        this.cancel();
-        this.mediator.trigger("block:focusPrevious", this.getStartBlock().blockID, { force: true });
-        return;
+    } else if (this.selecting) {
+      if (["Down", "ArrowDown"].indexOf(key) > -1) {
+        ev.preventDefault();
+        if (ev.shiftKey && ev.altKey) this.expandToEnd();
```

Comment:
Looks like you've caught all the different behaviours here :+1:

Reply:
Wondering if there are more key combos we haven't considered, especially on Windows. For example, does pgstrt key go to the start of the text input? (http://osxdaily.com/2015/06/17/home-end-button-mac-keyboard-functions/)

That key might not be relevant but there might be some others. This is great for now though

Reply:
Good point I'd say we get this in and then think about edge cases.